README.md

```diff
@@ -1,252 +1,40 @@
-    con.execute("INSERT INTO companies (name, num_employees) VALUES ('QuantCo', 100), ('Google', 150000), ('BMW', 120000), ('Apple', 145000)")
-    con.execute("CREATE TABLE companies_archive (id INTEGER PRIMARY KEY, name TEXT, num_employees INTEGER)")
-    con.execute("INSERT INTO companies_archive (name, num_employees) VALUES ('QuantCo', 90), ('Google', 140000), ('BMW', 110000)")
-```
+Please refer to the [Getting Started](https://datajugde.readthedocs.io/en/latest/getting_started.html) section of our documentation for details.
 
-The table "companies_archive" contains three entries:
-
-| id  | name    | num_employees |
-| --- | ------- | ------------- |
-| 1   | QuantCo | 90            |
-| 2   | Google  | 140,000       |
-| 3   | BMW     | 110,000       |
-
-While "companies" contains an additional entry:
-
-| id  | name    | num_employees |
-| --- | ------- | ------------- |
-| 1   | QuantCo | 100           |
-| 2   | Google  | 150,000       |
-| 3   | BMW     | 120,000       |
-| 4   | Apple   | 145,000       |
-
-As an example, we will run 4 tests on this table:
-
-- Does the table "companies" contain a column named "name"?
-- Does the table "companies" contain at least 1 entry with the name "QuantCo"?
-- Does the column "num_employees" of the "companies" table have all positive values?
-- Does the column "name" of the table "companies" contain at least all the values of
-  the corresponding column in "companies_archive"?
+Expressing an expectations between different tables from a database may look as such:
 
 ```python
-import pytest
-import sqlalchemy as sa
-
 from datajudge import (
-    Condition,
-    WithinRequirement,
-    BetweenRequirement,
-)
-from datajudge.pytest_integration import collect_data_tests
-
-
-# We create a Requirement (within table). This object will contain
-# all the constraints we want to test on the specified table.
-# To test another table or test the same table against another table,
-# we would create another Requirement object.
-companies_req = WithinRequirement.from_table(
-    db_name="example", schema_name=None, table_name="companies"
+    BetweenRequirement
 )
 
-# Adding a constraint: column "name" should exist in our table
-companies_req.add_column_existence_constraint(columns=["name"])
-
-# Adding a constraint with a condiction
-condition = Condition(raw_string="name = 'QuantCo'")
-companies_req.add_n_rows_min_constraint(n_rows_min=1, condition=condition)
-
-companies_req.add_numeric_min_constraint(column="num_employees", min_value=1)
-
-# Creating a requirement (set of constraints) to test constraint between
-# the table "companies" and the table "companies_archive".
 companies_between_req = BetweenRequirement.from_tables(
     db_name1="example",
-    schema_name1=None,
     table_name1="companies",
     db_name2="example",
-    schema_name2=None,
     table_name2="companies_archive",
 )
 
 companies_between_req.add_row_superset_constraint(
```

Comment:
It'd be nice to give some context here of what you try to convey with this constraint? Probably as a code comment or prose right before the code block.

Reply:
I tried to do so.